src/dist/replication/test/replica_test/unit_test/replica_disk_test.cpp

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2017-present, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+
+#include <gtest/gtest.h>
+
+#include <dsn/utility/fail_point.h>
+#include "replica_test_base.h"
+
+namespace dsn {
+namespace replication {
+
+class replica_disk_test : public testing::Test
+{
+public:
+    std::unique_ptr<mock_replica_stub> _stub;
+    int app_id = 1;
+    int dir_nodes_count = 2;
```

Comment:
Could you set some large value to this variable for test? like 7, 12.

Reply:
Yes, test 10~